org.nickvision.tagger.json

```
@@ -0,0 +1,89 @@
+{
+    "app-id" : "org.nickvision.tagger",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "42",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "org.nickvision.tagger",
+    "finish-args":[
+        "--socket=x11",
+        "--share=ipc",
```

Since you say it used GTK4, it should be:
```suggestion
        "--socket=fallback-x11",
        "--socket=wayland",
        "--device=dri",
        "--share=ipc",
```